lua/wire/wireshared.lua

```diff
@@ -941,6 +946,15 @@ function WireLib.setAng(ent, ang)
 	return ent:SetAngles(ang)
 end
 
+function WireLib.setLocalAng(ent, ang)
+	if isnan(ang.pitch) or isnan(ang.yaw) or isnan(ang.roll) then return end
+	if abs(ang.pitch) == huge or abs(ang.yaw) == huge or abs(ang.roll) == huge then return false end -- SetAngles'ing inf crashes the server
+
+	ang = Angle(ang)
```

Comment:
Not sure why this is in the WireLib.setAng function but it should not be needed here especially

Reply:
Which part of it is not needed?

Reply:
The ang = Angle(ang). Idk why github shows more lines than you select

Reply:
Nvm im stupid.

Reply:
> Although the nan / infinity checks are probably unneccessary too, but better to keep them for now just in case
I checked on my server and the game automatically stops it from setting the angle to inf so its probably good.

Reply:
Make sure `ang.pitch=math.huge` doesn't bypass that

Reply:
> Make sure `ang.pitch=math.huge` doesn't bypass that

When you try to rotate an entity with math.huge as one of the values it shows this in the console.
![image](https://github.com/wiremod/wire/assets/24782457/df4f12d6-7e9c-49f1-82f5-aed4b70c0ee8)